storage/store.go

```diff
@@ -938,27 +947,33 @@ func (s *Store) NewRangeDescriptor(start, end proto.Key, replicas []proto.Replic
 
 // SplitRange shortens the original range to accommodate the new
 // range. The new range is added to the ranges map and the rangesByKey
-// sorted slice.
+// btree.
 func (s *Store) SplitRange(origRng, newRng *Range) error {
 	if !bytes.Equal(origRng.Desc().EndKey, newRng.Desc().EndKey) ||
 		bytes.Compare(origRng.Desc().StartKey, newRng.Desc().StartKey) >= 0 {
 		return util.Errorf("orig range is not splittable by new range: %+v, %+v", origRng.Desc(), newRng.Desc())
 	}
-	// Replace the end key of the original range with the start key of
-	// the new range.
-	copyDesc := *origRng.Desc()
```

Comment:
i think this was accidentally renamed (https://github.com/cockroachdb/cockroach/pull/976/files#diff-dae9f8a58ca2ec436440b12989c1982dR965)